docs/registry/txt.md

```diff
@@ -95,3 +95,13 @@ The TXT registry can optionally cache DNS records read from the provider. This c
 rate limits imposed by the provider.
 
 Caching is enabled by specifying a cache duration with the `--txt-cache-interval` flag.
+
+## Owner migration
+
+It is possible to migrate the owner of your records when using the TXT registry.
+
+Simply use the following flags when setting up your instance : `--migrate-txt-owner`, which is the boolean enabling
```

Comment:
```suggestion
One can use the following flags when setting up your instance : `--migrate-txt-owner`, which is the boolean enabling
```
words like "simple" can be offensive. It's recommended to avoid them.